pages/date-range-picker/month-calendar-permutations.page.tsx

```diff
@@ -26,18 +26,34 @@ const intervals = [
 ];
 
 const permutations = createPermutations<DateRangePickerCalendarProps>([
+  // Selection range
   ...intervals.map(([startDate, endDate]) => ({
     value: [{ start: { date: startDate, time: '' }, end: { date: endDate, time: '' } }],
     setValue: [() => {}],
     locale: ['en-GB'],
     startOfWeek: [1],
-    isDateEnabled: [() => true, () => false, (date: Date) => date.getDate() % 2 !== 0],
     onChange: [() => {}],
     timeInputFormat: ['hh:mm:ss'] as const,
     i18nStrings: [i18nStrings],
-    dateOnly: [false, true],
     customAbsoluteRangeControl: [undefined],
   })),
+  // Disabled dates
```

Comment:
Non-blocking suggestion: Add h2 headers in between each permutation block to better map the parts to the code

Reply:
I did it for the slider permutations, but can't do here as easily due to the way those dropdowns are rendered. If I add the labels - it messes up with the elements placement.